packages/terra-notification-dialog/CHANGELOG.md

```diff
@@ -2,441 +2,230 @@
 
 ## Unreleased
 
-## 3.39.0 - (August 11, 2020)
```

Comment:
what the heck happened to this change log?

Reply:
3.39.0 has been released already:
https://www.npmjs.com/package/terra-notification-dialog

Reply:
I'll look at it again. I had a heck of a time with merge conflicts & then just copied & pasted from remote/main

Reply:
Ah looks like I just didn't get one fixed correctly... wow!

Reply:
fixed: https://github.com/cerner/terra-framework/pull/1105/commits/abb5364c1fa144602adab79f0c3c397faffab3f6